modules.d/99squash/module-setup.sh

```diff
@@ -5,7 +5,7 @@ check() {
 
     for i in CONFIG_SQUASHFS CONFIG_BLK_DEV_LOOP CONFIG_OVERLAY_FS ; do
         if ! check_kernel_config $i; then
-            dinfo "dracut-squash module requires kernel configuration $i (y or m)"
+            derror "dracut-squash module requires kernel configuration $i (y or m)"
```

Comment:
Please don't error here yet... We will introduce a special dcheck_error, which errors, if the modules was explicitly specified.

Reply:
OK, will update .

Reply:
still there

Reply:
@ryncsn you forgot this

Reply:
silly me, will fix it.